src/io/inputs.jl

```diff
@@ -1,6 +1,7 @@
-using JSON
-using YAXArrays
-using NetCDF
+using Distributions,
+    JSON,
+    NetCDF,
+    YAXArrays
```

Comment:
Minor note, can be ignored for now but in future I'd prefer imports to be grouped by conceptual purpose:

```
using Distributions
using
    JSON,
    NetCDF,
    YAXArrays